src/main/scala/mesosphere/marathon/core/appinfo/package.scala

```diff
@@ -0,0 +1,11 @@
+package mesosphere.marathon.core
+
+import mesosphere.marathon.core.pod.PodDefinition
+import mesosphere.marathon.state.{ AppDefinition, Group }
+
+package object appinfo {
+
+  type AppSelector = Selector[AppDefinition]
```

Comment:
happy to remove. did this initially to help with conversion to Selector[AppDefinition]